src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Encapsulates the Duke task manager bot.
+ *
+ * @author William Ming
```

Comment:
Overall coding standard and naming is well done, no major faults found. Keep up the good work!